src/decoder/Makefile

```diff
@@ -2,12 +2,20 @@ all:
 
 EXTRA_CXXFLAGS = -Wno-sign-compare
 include ../kaldi.mk
+LDFLAGS += $(CUDA_LDFLAGS)
+LDLIBS += $(CUDA_LDLIBS)
+LDLIBS += -lnvToolsExt -lcudart
+CUDAOPTS =
+#CUDAOPTS += -G -g 
 
 TESTFILES =
 
 OBJFILES = training-graph-compiler.o lattice-simple-decoder.o lattice-faster-decoder.o \
    lattice-faster-online-decoder.o simple-decoder.o faster-decoder.o \
-   decoder-wrappers.o
```

Comment:
is there a reason you're splitting these into several lines?

Reply:
done